erc/ERCS/erc-7562.md

```diff
@@ -348,11 +353,11 @@ These are storage rules that help to protect the bundler from denial of service:
     * **`opsSeen`, `opsIncluded`, and reputation calculation** are defined above.
     * `UnstakedReputation` of an entity determines the maximum number of entries using this entity allowed in the mempool.
     * `opsAllowed` is a reputation-based calculation for an unstaked entity, representing how many `UserOperations` it is allowed to have in the mempool.
```

Comment:
I think need a better babe for opsAllowed, eg opsAllowedInMempool (though mempool is confusing here wrt alt-mempools

Reply:
Babe == name I assume?

Reply:
I think that this is an OK name, given the context and the definition here..